src/NServiceBus.RabbitMQ/Administration/DelayInfrastructure.cs

```diff
@@ -8,6 +8,7 @@ static class DelayInfrastructure
     {
         const int maxLevel = 32 - 1;
         public const string DelayHeader = "NServiceBus.Transport.RabbitMQ.DelayInSeconds";
+        public const string DelayDestination = "nsb.delay-destination";
```

Comment:
`DelayDestinationExchangeName`? otherwise it can be confused with the actual destination